test/viewModels/DepartureStatusSpec.scala

```diff
@@ -25,26 +25,37 @@ import org.scalatestplus.scalacheck.ScalaCheckPropertyChecks
 class DepartureStatusSpec extends SpecBase with Generators with ScalaCheckPropertyChecks {
 
   "Departure Status" - {
-    val statusus = Seq(
-      ("DepartureSubmitted"           -> "departure.status.submitted"),
-      ("MrnAllocated"                 -> "departure.status.mrnAllocated"),
-      ("ReleasedForTransit"           -> "departure.status.releasedForTransit"),
-      ("TransitDeclarationRejected"   -> "departure.status.transitDeclarationRejected"),
-      ("DepartureDeclarationReceived" -> "departure.status.departureDeclarationReceived"),
-      ("GuaranteeValidationFail"      -> "departure.status.guaranteeValidationFail"),
-      ("TransitDeclarationSent"       -> "departure.status.transitDeclarationSent")
+    val statusOptions = Seq(
```

Comment:
@chris-ianson Do we need a test for `positiveAcknowledgement`?

Reply:
Absolutely, what was I thinking (good spot, I'll put it in)

Reply:
@PSouthern Updated